src/Nethermind/Nethermind.Consensus/Messages/BlockErrorMessages.cs

```diff
@@ -116,4 +117,7 @@ public static class BlockErrorMessages
 
     public const string NegativeGasUsed =
         "NegativeGasUsed: Cannot be negative.";
+    public static string MissingDeposits => "MissingDeposits: Deposits cannot be null in block when EIP-6110 activated.";
```

Comment:
Why do we have such messages for deposits in BlockErrorMessages but we don't for withdrawals and 7002?

Reply:
we have MissingWithdrawals Error, I am not sure abt Eip7002 tho